x11-misc/polybar/polybar-3.2.1.ebuild

```diff
@@ -1,10 +1,10 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 PYTHON_COMPAT=( python2_7 )
 
-inherit cmake-utils python-single-r1
+inherit python-single-r1
```

Comment:
cmake-utils is the eclass to use. The EAPI 7 support is work-in-progress, so you just need to wait for it; not work around it by regressing the ebuild.

Reply:
Understood, I will wait until it supports EAPI 7 and use that eclass.

Thanks you very much for the information